.pnp.js

```diff
@@ -180,7 +180,7 @@ function $$SETUP_STATE(hydrateRuntimeState, basePath) {
       }
     ],
     "enableTopLevelFallback": true,
-    "ignorePatternData": "(^(?:\\.vscode\\/pnpify(?:\\/(?!\\.)(?:(?:(?!(?:^|\\/)\\.).)*?)|$))$)",
+    "ignorePatternData": "(^(?:\\.vscode[\\\\\\/]pnpify(?:[\\\\\\/](?!\\.)(?:(?:(?!(?:^|[\\\\\\/])\\.).)*?)|$))$)",
```

Comment:
This is an interesting change, happens when I run `yarn`

Reply:
Weird - this string is generated by `micromatch` ([here](https://github.com/yarnpkg/berry/blob/master/packages/plugin-pnp/sources/PnpLinker.ts#L165)), I'm not sure why it would have changed 🤔